content/source/docs/enterprise/private/install-installer.html.md

```diff
@@ -110,6 +116,14 @@ On the Console Settings page, there is a section for HTTP Proxy:
 
 ![PTFE HTTP Proxy Settings](./assets/ptfe-http-proxy.png)
 
+#### Proxy Exclusions (NOPROXY)
```

Comment:
I think we've usually had this as `NO_PROXY` or `no_proxy` - is there a reason it's `NOPROXY` here?

(also applies to the migration page)